### PR TITLE
Add ruleset for thedailywtf.com

### PR DESCRIPTION
**Pull request by Pr0methean**

### CODE REVIEW

File: src/chrome/content/rules/thedailywtf.com.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<ruleset name="thedailywtf.com">
+	<target host="thedailywtf.com" />
+	<target host="*.thedailywtf.com" />
+
+	<test url="http://www.thedailywtf.com/" />
+  <test url="http://thedailywtf.com/articles/The_Brillant_Paula_Bean" />
```
Comment (pipboy96) [inline src/chrome/content/rules/thedailywtf.com.xml]: `s/  /\t/`

**Comment by pipboy96** [comment]
Do not use wildcards, there are only a few subdomains. Try these and add the working ones (and add non-functioning ones into a comment):

```
thedailywtf.com
www.thedailywtf.com
forums.thedailywtf.com
img.thedailywtf.com
live.thedailywtf.com
omg.thedailywtf.com
omg2.thedailywtf.com
cms.omg2.thedailywtf.com
what.thedailywtf.com
```

File: src/chrome/content/rules/thedailywtf.com.xml [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
+	<target host="what.thedailywtf.com" />
+	
 	<test url="http://thedailywtf.com/articles/The_Brillant_Paula_Bean" />
 	<test url="http://www.thedailywtf.com/articles/The_Brillant_Paula_Bean" />
-	<test url="http://what.thedailywtf.com/" />
 	<test url="http://what.thedailywtf.com/topic/20319/wtf-bites" />
-	<test url="https://what.thedailywtf.com/topic/20319/wtf-bites/52?lang=en-US&page=2" />
+	<test url="https://what.thedailywtf.com/topic/20319/wtf-bites/52?lang=en-US&amp;page=2" />
 	<rule from="^http://([\w-]+\.)?thedailywtf\.com/"
 		to="https://$1thedailywtf.com/" />
```
Comment (pipboy96) [inline src/chrome/content/rules/thedailywtf.com.xml]:
```
	<rule from="^http:" to="https:" />
```

File: src/chrome/content/rules/thedailywtf.com.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	<test url="http://thedailywtf.com/articles/The_Brillant_Paula_Bean" />
+	<test url="http://www.thedailywtf.com/" />
+	<test url="http://www.thedailywtf.com/articles/The_Brillant_Paula_Bean" />
+	<test url="http://what.thedailywtf.com/" />
+	<test url="http://what.thedailywtf.com/topic/20319/wtf-bites" />
+	<test url="http://what.thedailywtf.com/topic/20319/wtf-bites/52?lang=en-US&amp;page=2" />
+	<test url="http://omg2.thedailywtf.com/" />
+	
+	<rule from="^http://www\.thedailywtf\.com"
+		to="https://thedailywtf.com" />
```
Comment (pipboy96) [inline src/chrome/content/rules/thedailywtf.com.xml]: Don't forget the slashes at the end.

File: src/chrome/content/rules/thedailywtf.com.xml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<ruleset name="thedailywtf.com">
+	<!-- Failing:
+	<target host="forums.thedailywtf.com" />
+	<target host="img.thedailywtf.com" />
+	<target host="live.thedailywtf.com" />
+	<target host="omg.thedailywtf.com" />
+	<target host="cms.omg2.thedailywtf.com" />
+	-->
```
Comment (pipboy96) [inline src/chrome/content/rules/thedailywtf.com.xml]: Please write out the reasons why these targets fail.

Comment (Pr0methean) [reply src/chrome/content/rules/thedailywtf.com.xml]: Done.

**Comment by pipboy96** [comment]
I formatted the ruleset to match the current format:

```
<!--
	Timeout:
		- live

	Mismatched:
		- www (fixed by this ruleset)
		- forums (fixed by this ruleset)
		- img
		- omg
		- cms.omg2
-->
<ruleset name="TheDailyWTF.com">
	<target host="thedailywtf.com" />
	<target host="www.thedailywtf.com" />
	<target host="forums.thedailywtf.com" />
	<target host="omg2.thedailywtf.com" />
	<target host="what.thedailywtf.com" />

	<rule from="^http://forums\.thedailywtf\.com/"
		to="https://what.thedailywtf.com/" />

	<rule from="^http://www\.thedailywtf\.com/"
		to="https://thedailywtf.com/" />

	<rule from="^http:" to="https:" />
</ruleset>
```

Also please rename file to `TheDailyWTF.com.xml`. If you can, please also squash your changes into a single commit (I may help you to do that).

**Comment by Pr0methean** [comment]
This PR has been ready to merge for a week. Is there anything I can do to help speed it along?

**Comment by Bisaloo** [comment]
Thank you!

**Pull request merged**